plugins/woocommerce-admin/client/products/sections/product-shipping-section.tsx

```diff
@@ -166,8 +173,8 @@ export const ProductShippingSection: React.FC = () => {
 									'woocommerce'
 								) }
 							</p>
-							<div className="product-shipping-section__container">
-								<div>
+							<div className="product-shipping-section__dimensions__body">
```

Comment:
We typically use a similar class naming style to [BEM](https://getbem.com/naming/) so this class could either be:

`product-shipping-section__dimensions-body`

Or if dimensions is its own standalone block:

`product-shipping-section-dimensions__body`

Reply:
Done